packages/commonwealth/client/scripts/state/api/threads/fetchThreads.ts

```diff
@@ -51,7 +51,6 @@ interface FetchBulkThreadsProps extends CommonProps {
 
 interface FetchThreadCountProps extends CommonProps {
   queryType: typeof QueryTypes.COUNT;
-  limit?: number;
```

Comment:
most db systems keep counts in the stats table, this shouldn't be inefficient unless PG is the exception

Reply:
Maybe this would be better replaced by a `threadExists` type of query (both framework-wise and Postgres)

Reply:
I think PG might be doing a full index scan rather than using the stats table...

Reply:
Like the idea of a different query to check if a community has threads, but wondering if we can enhance the existing GetCommunity query with this flag instead... in any case, this is a rare edge case in the admin page, probably not worth the extra effort

Reply:
Actually, I'm taking a second look at the code and seems like we don't even need this query at all, since communities have a lifetime_thread_count property that can be used instead